app/src/androidTest/java/com/github/se/cyrcle/ui/testEnd2End/MainActivityTest.kt

```diff
@@ -127,6 +133,52 @@ class MainActivityTest {
     mapRobot.assertIsOfflineMode()
   }
 
+  @Test
+  fun testReview() {
+
+    composeTestRule.activity.userRepository.addUser(TestInstancesUser.user1, {}, {})
+    composeTestRule.activity.reviewRepository.addReview(TestInstancesReview.review6, {}, {})
+
+    // Authenticated user
+    authRobot.assertAuthScreen()
+    authRobot.performSignIn()
+
+    mapRobot.assertMapScreen()
+    mapRobot.toList()
+
+    listRobot.assertListScreen()
+    listRobot.toCard(0)
+
+    cardRobot.assertParkingDetailsScreen()
+    cardRobot.goAllReviews()
+
+    allReviewsRobot.assertAllReviewScreen(true)
+    allReviewsRobot.goToAddReview()
+
+    addReviewRobot.assertAddReviewScreen()
+    addReviewRobot.addReview()
```

Comment:
Do you think that you could expand the test such that you also remove the review later on ?